src/main.tsx

```diff
@@ -2,8 +2,37 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 import App from "./App.js";
 
+import { createGlobalStyle } from 'styled-components';
+
+const GlobalStyle = createGlobalStyle`
```

Comment:
I'd define GlobalStyle elsewhere. We can create a design system that will contain all the logic for styling and theming.

Reply:
Great for variables and resets, but I would not set there everything, have all styles at one file will be chaotic and miss the point of styled components